cmd/foreach/foreach.go

```diff
@@ -35,15 +35,19 @@ import (
 var exec executor.Executor = executor.NewRealExecutor()
 
 var (
-	repoFile = "repos.txt"
+	repoFile   string
+	successful bool
+	failed     bool
 
 	overallResultsDirectory string
 
 	successfulResultsDirectory string
 	successfulReposFileName    string
+	successfulReposSymlink     = ".latest_successful"
```

Comment:
Do we definitely need two symlinks? If we have our output directories to be something like:

```
some_temp_dir_from_os/
    some_random_bit/
        successful/
            repos.txt
            logs/
        failed/
            repos.txt
            logs/
```

Then we could just use one symlink, to the `/some_temp_dir_from_os/some_random_bit` path and infer the rest as needed.